.mailmap

```diff
@@ -561,6 +561,8 @@ Evandro Bernardes <evbernardes@gmail.com> Evandro <15084103+evbernardes@users.no
 Evandro Bernardes <evbernardes@gmail.com> Evandro Bernardes <s.evandro@hotmail.com>
 Evandro Bernardes <evbernardes@gmail.com> evbernardes <s.evandro@hotmail.com>
 Evani Balasubramanyam <balasubramanyam.evani@gmail.com> evani balasubramanyam <balasubramanyam.evani@gmail.com>
+Evelyn King <evelyn.cameron.king@gmail.com>
+Evelyn King <evelyn.cameron.king@gmail.com> Cameron King <v-caking@microsoft.com>
```

Comment:
Only the second line here is needed and also the other line saying Cameron King should be removed.